Gemfile.lock

```diff
@@ -263,7 +263,7 @@ GEM
     selenium-webdriver (3.142.3)
       childprocess (>= 0.5, < 2.0)
       rubyzip (~> 1.2, >= 1.2.2)
-    sentry-raven (2.11.0)
+    sentry-raven (2.11.1)
```

Comment:
We should probably not include this change as part of this release? Or add to the change notes?

Reply:
Traditionally, part of the release process we run `bundle install` which by default updates all the dependencies. You can see how this was bumped in previous releases: https://github.com/alphagov/govuk_publishing_components/pull/1040/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R271

Extract from [Publishing to RubyGems](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/publishing-to-rubygems.md) doc: _Run bundle install && npm install to ensure you have the latest dependencies installed._